tests/python_client/milvus_client/test_milvus_client_insert.py

```diff
@@ -1342,5 +1344,768 @@ def test_milvus_client_insert_after_json_path_index(self, enable_dynamic_field,
+
+            self.upsert(client, collection_name, new_rows, partial_update=True)
+            result = self.query(client, collection_name, filter=f"{primary_key_field_name} >= 0",
+                    check_task=CheckTasks.check_query_results,
+                    output_fields=[update_field_name],
+                    check_items={exp_res: new_rows,
+                                "with_vec": True,
+                                "vector_type": schema.fields[i if i != 0 else 1].dtype,
+                                "vector_field": update_field_name,
+                                "pk_name": primary_key_field_name})[0]
+            assert len(result) == default_nb
+
+        self.drop_collection(client, collection_name)
+    
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Partial update not ready")
+    def test_milvus_client_partial_update_new_field_with_dynamic_field(self):
+        """
+        target:  Test PU will success when provided empty data
+        method:
+            1. Create a collection
+            2. partial upsert new field
+        expected: Step 2 should result success
+        """
+        # step 1: create collection
+        client = self._client()
+        schema = self.create_schema(client, enable_dynamic_field=True)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
+        schema.add_field(default_vector_field_name, DataType.FLOAT_VECTOR, dim=default_dim)
+        index_params = self.prepare_index_params(client)[0]
+        index_params.add_index(default_primary_key_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_vector_field_name, index_type="AUTOINDEX")
+        collection_name = cf.gen_collection_name_by_testcase_name(module_index=1)
+        self.create_collection(client, collection_name, default_dim, schema=schema, 
+                               consistency_level="Strong", index_params=index_params)
+        
+        # step 2: partial upsert new field
+        rows = cf.gen_row_data_by_schema(nb=default_nb, schema=schema)
+        self.upsert(client, collection_name, rows, partial_update=True)
+        new_rows = [{default_primary_key_field_name: i, default_int32_field_name: 99} for i in range(default_nb)]
+        self.upsert(client, collection_name, new_rows, partial_update=True)
+
+        self.query(client, collection_name, filter=default_search_exp,
+                check_task=CheckTasks.check_query_results,
+                output_fields=[default_int32_field_name],
+                check_items={exp_res: new_rows,
+                                "with_vec": True,
+                                "pk_name": default_primary_key_field_name})[0]
+        
+        self.drop_collection(client, collection_name)
+    
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Partial update not ready")
+    def test_milvus_client_partial_update_valid_data(self):
```

Comment:
I think we'd better try to update every type of supported scalar field in milvus